Apache-Beam-Jobs.md

```diff
@@ -1,51 +1,39 @@
 ## Table of contents
 
-* [Introduction](#introduction)
-* [Running Apache Beam Jobs](#running-apache-beam-jobs)
-* [Apache Beam Job Architecture](#apache-beam-job-architecture)
-  * [`Pipeline`s](#pipelines)
-  * [`PValue`s](#pvalues)
-  * [`PTransform`s](#ptransforms)
-    * [`ParDo` and `DoFn`](#pardo-and-dofn)
-    * [`Map` and `FlatMap`](#map-and-flatmap)
-    * [`Filter`](#filter)
-    * [`GroupByKey`](#groupbykey)
-    * [Example of using `GroupByKey`,`Filter`, and `FlatMap`](#example-of-using-groupbykeyfilter-and-flatmap)
-  * [`Runner`s](#runners)
-* [Case studies](#case-studies)
-  * [Case study: `CountAllModelsJob`](#case-study-countallmodelsjob)
-  * [Case Study: `SchemaMigrationJob`](#case-study-schemamigrationjob)
+-   [Introduction](#introduction)
+-   [Apache Beam Job Architecture](#apache-beam-job-architecture)
+    -   [`Pipeline`s](#pipelines)
+    -   [`PValue`s](#pvalues)
+    -   [`PTransform`s](#ptransforms)
+        -   [`ParDo` and `DoFn`](#pardo-and-dofn)
+        -   [`Map` and `FlatMap`](#map-and-flatmap)
+        -   [`Filter`](#filter)
+        -   [`GroupByKey`](#groupbykey)
+            -   [Example of using `GroupByKey`,`Filter`, and `FlatMap`](#example-of-using-groupbykeyfilter-and-flatmap)
+    -   [`Runner`s](#runners)
+-   [Writing Apache Beam Jobs](#writing-apache-beam-jobs)
+-   [Testing Apache Beam Jobs](#testing-apache-beam-jobs)
+-   [Running Apache Beam Jobs](#running-apache-beam-jobs)
+-   [Case studies](#case-studies)
+    -   [Case Study: `SchemaMigrationJob`](#case-study-schemamigrationjob)
```

Comment:
Can you preserve the original format (e.g. with asterisks, not so many spaces after each bullet)? The original table of contents was auto-generated

Reply:
Done.